scripts/qpcr_dilution_calculation.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+import sys
+
+from EPPs.common import StepEPP, InvalidStepError
+
+
+class QPCRDilutionCalculation(StepEPP):
```

Comment:
If this script is not needed anymore then, we should remove it.

Reply:
removed